homeassistant/components/media_player/__init__.py

```diff
@@ -812,6 +837,22 @@ def state_attributes(self):
 
         return state_attr
 
+    async def async_browse_media(self, media_content_type=None, media_content_id=None):
+        """
+        Return a payload for the "media_player/browse_media" websocket command.
+
+        Payload should follow this format:
+            [{
```

Comment:
We should always describe the folder that we're in. So have it follow the same format and have an `items` key.

```json
{
  "title": str - Title of the item
  "thumbnail": str - URL to image thumbnail for item if available
  "media_content_id": str - Can be used as-is to send to media_player.play_media
  "media_content_type": str - Can be used as-is to send to media_player.play_media
  "can_play": bool - If item is playable
  "can_expand": bool - If item contains other media
  "children": [
    {
      "title": str - Title of the item
      "thumbnail": str - URL to image thumbnail for item if available
      "media_content_id": str - Can be used as-is to send to media_player.play_media
      "media_content_type": str - Can be used as-is to send to media_player.play_media
      "can_play": bool - If item is playable
      "can_expand": bool - If item contains other media
    },
  ]
}```